RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -379,14 +432,22 @@ public class RileyLinkDeviceTableViewController: UITableViewController {
                 }
             case .PressDownButton:
                 vc = CommandResponseViewController { [unowned self] (completionHandler) -> String in
-                    self.device.ops?.pressButton({ (response) in
-                        dispatch_async(dispatch_get_main_queue()) {
-                            switch response {
-                            case .Success(let msg):
-                                completionHandler(responseText: "Result: " + msg)
-                            case .Failure(let error):
-                                completionHandler(responseText: String(error))
-                            }
+//                    self.device.ops?.pressButton({ (response) in
```

Comment:
Yikes!  Yeah, I wasn't seeing any temp basals being enacted in Loop, so I wanted to try in RL, so I could see what was going on.  Turned out I had increased my max basal in Loop, but not on the pump, so it was failing. Thanks so much for catching this!